File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/twitter/SampleListTwitterActivity.java
package com.jdroid.android.sample.ui.twitter;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class SampleListTwitterActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return SampleListTwitterFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/twitter/SampleListTwitterActivity.kt
package com.jdroid.android.sample.ui.twitter

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class SampleListTwitterActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return SampleListTwitterFragment::class.java
    }
}
